types/react-reconciler/index.d.ts

```diff
@@ -412,12 +412,12 @@ declare namespace ReactReconciler {
         /**
          * This method should make the `instance` visible, undoing what `hideInstance` did.
          */
-        unhideInstance?(instance: Instance): void;
+        unhideInstance?(instance: Instance, props?: Props): void;
 
         /**
          * Same as `unhideInstance`, but for nodes created by `createTextInstance`.
          */
-        unhideTextInstance?(textInstance: TextInstance): void;
+        unhideTextInstance?(textInstance: TextInstance, text?: string): void;
```

Comment:
Do the new parameters need to be optional?

Reply:
Yes, you are right, it shouldn't